modules/core/include/opencv2/core/mat.hpp

```diff
@@ -2110,7 +2110,7 @@ class CV_EXPORTS Mat
          - number of channels
      */
     int flags;
-    //! the matrix dimensionality, >= 2
+    //! the matrix dimensionality, >= 2 (can also be 0 when the array is empty)
```

Comment:
`Mat` is designed like a smart pointer over buffer.

We can't access of any attributes of empty `Mat` - it is undefined behavior.

Reply:
I'm not sure I understand.

As the commit message says, I'm copying an existing sentence from elsewhere in the same documentation page. Is that existing sentence wrong?